shapes/typeIndex.shacl

```diff
@@ -0,0 +1,25 @@
+PREFIX sh: <http://www.w3.org/ns/shacl#>
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+PREFIX solid: <http://www.w3.org/ns/solid/terms#>
+PREFIX : <#>
+
+:TypeIndexShape a sh:NodeShape ;
```

Comment:
Maybe rename this to something different, e.g. :TypeIndexDocumentShape. If I understand https://solid.github.io/type-indexes/#public-type-index-example correctly, the subject here should be something different than the subject on line 9?